src/ghost/ghost-post.service.integration.test.ts

```diff
@@ -380,6 +386,90 @@ describe('GhostPostService', () => {
         });
     });
 
+    describe('createGhostPost', () => {
+        it('should create a new ghost post successfully', async () => {
```

Comment:
Hm, I was thinking that maybe a unit test would be better for this and then a separate integration test for the mapping insertion - Really `createGhostPost` is just a wrapper around the post service create post method, and that should already have tests, so in a unit test we could just test that this dep gets called with the correct params (meaning we don't need to duplicate the testing of the db insertions etc). What do you think?